zproject/test_settings.py

```diff
@@ -125,6 +125,7 @@ def set_loglevel(logger_name, level) -> None:
     set_loglevel('zulip.send_email', 'ERROR')
     set_loglevel('zerver.lib.digest', 'ERROR')
     set_loglevel('zerver.lib.email_mirror', 'ERROR')
+    set_loglevel('zerver.worker.queue_processors', 'INFO')
```

Comment:
Huh. I'm surprised that this does anything -- our default loglevel is already INFO.

... Oh wow. It's that that `set_loglevel` helper is completely bogus -- what it actually does is suppress logs from that logger *unconditionally*, unless it's a logger that we've configured a custom set of handlers for. Which this isn't.

I guess I'll go fix that.

Reply:
OK, done, as cd5e44491 and ancestors.

You'll want to set this to something like WARNING, because this helper now actually does what it says on the tin. :-)